runtime/oti/j9nonbuilder.h

```diff
@@ -4252,6 +4252,7 @@ typedef struct J9JITConfig {
 	uint64_t clientUID;
 	uint64_t serverUID;
 #endif /* J9VM_OPT_JITSERVER */
+        void (*jitAddNewLowToHighRSSRegion)(const char *name, uint8_t *start, uint32_t size, size_t pageSize);
```

Comment:
This should use a tab rather than spaces.

Reply:
Fixed